styles/planet-minecraft/catppuccin.user.css

```
@@ -128,6 +128,32 @@
       }
     }
 
+    .content-actions {
+      li {
+        background: @surface0 !important;
+        border-color: @surface0 !important;
+
+        a {
+          color: @text !important;
+        }
+
+        a:hover,
+        .js_link:hover,
+        .link:hover {
+          background-color: @surface2 !important;
+          color: @text !important;
+
+          .material-icons {
+            color: @accent-color !important;
+          }
+        }
+
+        .material-icons {
+          color: @text !important;
```

```suggestion
          color: @subtext0 !important;
```
Please stop changing the colors. It was `subtext0` originally and I don't see a reason to change it.

---

i accidentally changed that while testing the css, sorry